front/app/components/PostShowComponents/Comments/messages.ts

```diff
@@ -163,19 +163,19 @@ export default defineMessages({
     id: 'app.components.Comments.mostRecent',
     defaultMessage: 'Most recent',
   },
-  mostUpvoted: {
-    id: 'app.components.Comments.mostUpvoted',
-    defaultMessage: 'Most voted',
+  mostLiked: {
+    id: 'app.components.Comments.mostLiked',
+    defaultMessage: 'Most reactions',
   },
-  upvoteComment: {
-    id: 'app.components.Comments.upvoteComment',
-    defaultMessage: 'Upvote this comment',
+  likeComment: {
+    id: 'app.components.Comments.likeComment',
+    defaultMessage: 'Like this comment',
   },
-  commentUpvote: {
-    id: 'app.containers.Comments.commentUpvote',
-    defaultMessage: 'Upvote',
+  commentLike: {
+    id: 'app.containers.Comments.commentLike',
+    defaultMessage: 'Like',
   },
-  commentCancelUpvote: {
+  commentCancelLike: {
     id: 'app.containers.Comments.commentCancelUpvote',
```

Comment:
Should we change the ID to match as well here?

Reply:
Ah nevermind actually - it's good to avoid re-translating things if we don't need to :) :+1:

Reply:
Yes, I've tried to avoid changing keys for copy that hasn't changed